cmd/webhook/main.go

```diff
@@ -91,9 +92,12 @@ func NewDefaultingAdmissionController(ctx context.Context, cmw configmap.Watcher
 	channelStore := channeldefaultconfig.NewStore(logging.FromContext(ctx).Named("channel-config-store"))
 	channelStore.WatchConfigs(cmw)
 
+	featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"))
+	featureStore.WatchConfigs(cmw)
+
 	// Decorate contexts with the current state of the config.
 	ctxFunc := func(ctx context.Context) context.Context {
-		return channelStore.ToContext(store.ToContext(ctx))
+		return featureStore.ToContext(channelStore.ToContext(store.ToContext(ctx)))
```

Comment:
that looks ridiculous. Are we really going to wrap wrap wrap `ToContext` calls like this?

Reply:
:man_shrugging: I think modifying this API is outside the scope of this PR...